README.md

```diff
@@ -38,25 +38,44 @@ cd /home/rstudio/AutoGVP
 1. Run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v0.4.4/docs/GERMLINE_SNV_ANNOT_README.md) first.
 This workflow currently annotates variants with ClinVar (2022-05-07).
 2. Run the [Pathogenicity Preprocessing Workflow](https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing), which performs ANNOVAR with InterVar and AutoPVS1 annotations.
-3. Run AutoGVP.
+3. Download required db files
```

Comment:
I think this can use more explanation and specificity. Eg: Download ClinVar variant and submission files, which will be used to .....